docs/server/quick-start/upgrade-guide.md

```diff
@@ -11,7 +11,15 @@ You can also download the package files for each platform from our [website](htt
 
 There is no longer a distinction between the open-source (OSS) and commercial versions of EventStoreDB. This unified release is licensed under [ESLv2](https://github.com/EventStore/EventStore/blob/master/LICENSE.md), meaning that anyone can access and use it, but enterprise features are only enabled with a valid license key.
 
-If you have a previous version of EventStoreDB installed through PackageCloud or Chocolatey, please uninstall those versions before installing version 24.10 from Cloudsmith.
+Packages for KurrentDB are still hosted on [Cloudsmith](https://cloudsmith.io/~eventstore), in the following repositories:
+
```

Comment:
The changes to this file shouldn't be cherry-picked across to 24.10 as this version isn't hosted in the new kurrentdb-* repositories on Cloudsmith.

The reference to Debian further up this file should be changed to `Ubuntu` though

Reply:
Got it. I'll revert all changes from the upgrade-guide.md file